### PR TITLE
import method with test passing against ephemeral connector

### PR DESCRIPTION
**Pull request by bkaptijn**

### CODE REVIEW

File: test/index.spec.js [hunk 1]
```diff
@@ -281,6 +282,36 @@ describe('desciple-core-api', () => {
...
+
+      // reset ephemeral connector (in memory mode)
+      let ConnectorModuleClass = await loadConnector('ephemeral')
+      discipl.registerConnector('ephemeral', new ConnectorModuleClass())
+
+      let result = await discipl.importLD({ ...ld, ...ld2 })
+      expect(result).to.equal(true)
+
+      let channelData = await discipl.exportLD(ssid)
+      expect(channelData[ssid.did][0][Object.keys(channelData[ssid.did][0])]).to.deep.equal({ 'need': 'food' })
```
Comment (pimotte) [inline test/index.spec.js]: I would consider maybe testing the entire data structure in one go. This is quite hard to read.

File: test/index.spec.js [hunk 2]
```diff
@@ -281,6 +282,36 @@ describe('desciple-core-api', () => {
       expect(exportedData[ssid.did][1]).to.deep.equal({ [claimlink2]: { 'need': 'wine' } })
       expect(exportedData[ssid.did][2]).to.deep.equal({ [attestationLink]: { 'agree': { [ssid2.did]: [ { [claimlink3]: { 'need': 'water' } } ] } } })
     })
+
+    it('should be able to import multiple verifiable claims in multiple channels in order (in ephemeral connector)', async () => {
+      let ssid = await discipl.newSsid('ephemeral')
+      let ssid2 = await discipl.newSsid('ephemeral')
+
+      await discipl.claim(ssid, { 'need': 'food' })
```
Comment (pimotte) [inline test/index.spec.js]: Do we need tests with respect to links that are being followed with attests? I'm not sure whether this produces special cases.

Comment (bkaptijn) [reply test/index.spec.js]: No, links can just be imported as object of the claim as attestation. They can point to anywhere and this does not have to be checked at import.

File: test/index.spec.js (hunk 1 above)
Comment (bkaptijn) [reply test/index.spec.js]: ok, weĺl need to improve on how to handle these JSON exports to get more readable code, not only within tests in which you can check against the whole datastructure.

**Pull request merged**